lte/gateway/c/core/oai/tasks/mme_app/mme_config.c

```diff
@@ -1887,11 +1907,11 @@ void mme_config_display(mme_config_t* config_pP) {
       bdata(config_pP->itti_config.log_file));
   OAILOG_INFO(LOG_CONFIG, "- SCTP:\n");
   OAILOG_INFO(
-      LOG_CONFIG, "    in streams .......: %u\n",
-      config_pP->sctp_config.in_streams);
+      LOG_CONFIG, "  upstream_sctp_sock..: %s\n",
+      bdata(config_pP->sctp_config.upstream_sctp_sock));
```

Comment:
we should free the according added bstrings on `free_mme_config` function

Reply:
Seems I did not push my most recent branchy - this issue was actually detected by my unit test when I added it. Pushing the latest branch which does as you suggest (nice review!)